### PR TITLE
Use ona utils

### PR DESCRIPTION
**Pull request by kahummer**
fix: #739

### CODE REVIEW

File: src/components/AssignTeamPopover/index.tsx [hunk 1]
```diff
@@ -41,6 +41,7 @@ const AssignTeamPopover = (props: AssignTeamPopoverProps) => {
 
   // define props for the actual popover component
   const popoverProps = {
+    flip: true,
```
Comment (moshthepitt) [inline src/components/AssignTeamPopover/index.tsx]: Why is this change in this PR?

File: src/gisida.d.ts [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
+  import { Dictionary } from '@onaio/utils';
   import { Map as mbMap } from 'mapbox-gl';
   import { ParseResult } from 'papaparse';
   import { AnyAction } from 'redux';
 
   /** interface for generic object */
-  interface FlexObject {
+  interface Dictionary {
     [key: string]: any;
   }
```
Comment (moshthepitt) [inline src/gisida.d.ts]: This looks like it should be removed, thoughts?

Comment (kahummer) [reply src/gisida.d.ts]: yeah for sure missed that

File: src/components/AssignTeamPopover/index.tsx (hunk 1 above)
Comment (kahummer) [reply src/components/AssignTeamPopover/index.tsx]: My bad It shouldn't be here

**Pull request merged**